apps/contracts/contracts/abstracts/TokenURISchema.sol

```diff
@@ -7,7 +7,7 @@ abstract contract TokenURISchema is ITokenURISchema {
     string private _schemaURI;
     mapping(uint256 => bytes) private _tokenURIDigests;
     bytes private _schemaDigest;
-    string private _hashAlgorithm = "sha256";
+    string private _hashAlgorithm = "keccak256";
```

Comment:
yeah, that is an important point. EVM uses a specific implementation of sha256, you already figured it out. :)